### PR TITLE
Add character title translations

### PR DESCRIPTION
**Pull request by robflop**
As the title says.

Fitting the translation for Matikanefukukitaru's title ingame feels like a lost cause, considering it's a massive kanji compound in JP, which expands a lot when translated into english.

### CODE REVIEW

**Review by stardustichi — APPROVED** [review]

File: src/data/uma-title.csv [hunk 1]
```diff
@@ -1,35 +1,35 @@
...
-"[あぶそりゅーと☆LOVE]",""
-"[Nevertheless]",""
-"[うららん一等賞♪]",""
-"[運気上昇☆幸福万来]",""
-"[ポインセチア・リボン]",""
-"[キング・オブ・エメラルド]",""
+"[スペシャルドリーマー]","[Special Dreamer]"
+"[サイレントイノセンス]","[Silent Innocence]"
+"[トップ・オブ・ジョイフル]","[Top of Joyful]"
+"[ビヨンド・ザ・ホライズン]","[Beyond The Horizon]"
```
Comment (gertasik) [inline src/data/uma-title.csv:5]: I've noticed that you used "the" in the other batch of titles, but now it's "The" with a capital T. Not sure if it's on purpose here, just thought I'd bring your attention to it

Other than that, looks fine to me

Comment (robflop) [reply src/data/uma-title.csv:5]: Tried to look where this is but can't find any examples right now. Could you show me some?

Comment (gertasik) [reply src/data/uma-title.csv:5]: "[輝く景色の、その先に]","Beyond the Brilliant Scenery"
"[見習い魔女と長い夜]","The Apprentice Witch and the Long Night"
"[テッペンに立て！]","Stand at the Top!"
"[不沈艦の進撃]","Advance of the Unsinkable Battleship"

Comment (robflop) [reply src/data/uma-title.csv]: Thanks. Fixed!

**Review by gertasik — APPROVED** [review]

**Pull request merged**